source/includes/language-compatibility-table-php.rst

```diff
@@ -6,12 +6,12 @@
    :class: compatibility-large
 
    * - PHP Driver
-     - PHP 5.5
-     - PHP 5.6
-     - PHP 7.0
-     - PHP 7.1
-     - PHP 7.2
      - PHP 7.3
+     - PHP 7.2
+     - PHP 7.1
+     - PHP 7.0
+     - PHP 5.6
+     - PHP 5.5
```

Comment:
Extension 1.6 will not have support for PHP 5.5 (see: [PHPC-1377](https://jira.mongodb.org/browse/PHPC-1377)).

I realize we don't report language compatibility for PHPLIB (although I suggest doing so in [DOCSP-6259](https://jira.mongodb.org/browse/DOCSP-6259)), but library version 1.5 also drops support for PHP 5.5 ([PHPLIB-429](https://jira.mongodb.org/browse/PHPLIB-429)).

Reply:
I've made significant updates to the PHP pages to incorporate the other ticket in JIRA